caldav-integration-webapp/src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<portlet-app xmlns="http://java.sun.com/xml/ns/portlet/portlet-app_2_0.xsd" version="2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://java.sun.com/xml/ns/portlet/portlet-app_2_0.xsd http://java.sun.com/xml/ns/portlet/portlet-app_2_0.xsd">
+  <portlet>
+    <portlet-name>CaldavIntegration</portlet-name>
```

Comment:
```suggestion
    <portlet-name>Caldav</portlet-name>
```

Reply:
Should caldavPortlet be replaced by caldav?